client/src/components/Content.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "styled-components";
+import Bundle from "./bundle/bundle";
+
+const SectionTitle = styled.div`
+    font-family: 'Sofia Pro', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    color: white;
+    font-size: 1.17em;
+    font-weight: bold;
+    margin: 1em;
+}
```

Comment:
You have an extra `}` character that is not needed. Remove this line

Reply:
Resolved